ckanext/push_errors/blueprints/push_errors.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from flask import Blueprint
+from ckan.plugins import toolkit
+
+log = logging.getLogger(__name__)
+
+push_error_bp = Blueprint('push_errors', __name__)
+
+
+@push_error_bp.route('/push-error/test')
```

Comment:
```suggestion
push_error_bp = Blueprint('push_errors', __name__, url_prefix='/push-error')


@push_error_bp.route('/test', methods=["GET"])
```
The prefix at the top and the invidual URLs is better
Define the allowed method in each view it's good